modules/drivers/sqlserver/src/metabase/driver/sqlserver.clj

```diff
@@ -969,3 +1008,12 @@
   [driver conn-spec schema]
   (let [sql [[(format "IF NOT EXISTS (SELECT * FROM sys.schemas WHERE name = '%s') EXEC('CREATE SCHEMA [%s];');" schema schema)]]]
     (driver/execute-raw-queries! driver conn-spec sql)))
+
+(defmethod driver/rename-tables!* :sqlserver
+  [_driver db-id sorted-rename-map]
+  (jdbc/with-db-transaction [conn (sql-jdbc.conn/db->pooled-connection-spec db-id)]
```

Comment:
Pretty sure SQLServer renames are not transactional. Can we rather use a regular connection and add a TODO around locking etc?